docs/self-managed/operational-guides/update-guide/keycloak/keycloak-update.md

```diff
@@ -0,0 +1,13 @@
+---
+id: keycloak-update
+title: Update Keycloak
+description: "Review what has to be taken into account when updating Keycloak."
+---
+
+When updating Keycloak, follow the [Keycloak upgrade guide](https://www.keycloak.org/docs/latest/upgrading/index.html) and refer to the [supported environments](reference/supported-environments.md#camunda-8-self-managed) to ensure compatibility with tested Keycloak versions.
+
+:::danger
+When updating Keycloak, ensure that you carry along its existing database.
+**Do not** update by creating a new Keycloak instance and re-importing your users from external sources (e.g. LDAP) as this will result in new Keycloak-internal ids.
+Otherwise, users may not be able to access their data (e.g. Optimize collections) or log in to Web Modeler (see [Web Modeler's log in troubleshooting guide](docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-login.md#unique-constraint-violation)) after the update as their Keycloak-internal ids may change.
```

Comment:
The last part seems a bit redundant, as you already wrote in the previous sentence that the Keycloak-internal IDs will change.
```suggestion
Otherwise, users may not be able to access their data (e.g. Optimize collections) or log in to Web Modeler (see [Web Modeler's login troubleshooting guide](docs/self-managed/modeler/web-modeler/troubleshooting/troubleshoot-login.md#unique-constraint-violation)) after the update.
```